esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/metoffice.cylc

```diff
@@ -6,7 +6,12 @@
             --wckey = RTW
             --ntasks = {{ MAX_PARALLEL_TASKS }}
 
-    [[process<metric=radiation_budget>]]
+    [[process<small>]]
         [[[directives]]]
-            --time = 2
-            --mem = 3G
+            --time = 4
+            --mem = 6G
+
+    [[process<medium>]]
+        [[[directives]]]
+            --time = 8
+            --mem = 6G
```

Comment:
Looking at the `job.time` files when running the RTW:

- process_ensclus
  ```
  Elapsed (wall clock) time (h:mm:ss or m:ss): 3:28.38
  Maximum resident set size (kbytes): 1140660  # (~1.1G)
  ```
- process_heatwaves_coldwaves:
  ```
  Elapsed (wall clock) time (h:mm:ss or m:ss): 1:59.48
  Maximum resident set size (kbytes): 1431116  # (~1.4G)
  ```
- process_modes_of_variability: 
  ```
  Elapsed (wall clock) time (h:mm:ss or m:ss): 1:06.10
  Maximum resident set size (kbytes): 1067240  # (~1.0G)
  ```
- process_multimodel_products:
  ```
  Elapsed (wall clock) time (h:mm:ss or m:ss): 0:56.42
  Maximum resident set size (kbytes): 1134544  # (~1.1G)
  ```
I would instead suggest the following (to avoid requesting resources we won't use):
```suggestion
            --time = 2
            --mem = 2G

    [[process<medium>]]
        [[[directives]]]
            --time = 4
            --mem = 2G
```
(these values work for me)

Reply:
See a7567301116d33ee36b65d069aede36852f28dca - these values work in repeated testing, any lower times occasionally fail with a timeout